src/Tools/Console/Command/MappingDescribeCommand.php

```diff
@@ -115,7 +168,7 @@ private function displayEntity(
                     $this->formatField('Embedded class?', $metadata->isEmbeddedClass),
                     $this->formatField('Parent classes', $metadata->parentClasses),
                     $this->formatField('Sub classes', $metadata->subClasses),
-                    $this->formatField('Embedded classes', $metadata->subClasses),
+                    $this->formatField('Embedded classes', $metadata->embeddedClasses),
```

Comment:
This should go to a separate commit. If it's a bugfix, it should even go to a separate branch.

Reply:
Ok, I made the PR here instead  :  https://github.com/doctrine/orm/pull/12078